pubspec.lock

```diff
@@ -189,7 +203,14 @@ packages:
       name: fake_async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.1.0-nullsafety"
+    version: "1.1.0"
+  ffi:
+    dependency: transitive
+    description:
+      name: ffi
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "0.1.3"
```

Comment:
This will break the web build. I added a dependency override for `path_provider` (see issue here https://github.com/flutter/flutter/issues/66143 ) to avoid this transitive dependency and it works fine on my flutter build, we need to figure out why your dependencies turn out different. @gabrielGavriluta has the same issue.